providers/standard/provider.yaml

```diff
@@ -93,3 +93,21 @@ config:
         type: string
         example: uv
         default: auto
+
+task-decorators:
+  - class-name: airflow.providers.standard.decorators.bash.bash_task
+    name: bash
+  - class-name: airflow.providers.standard.decorators.branch_external_python.branch_external_python_task
+    name: branch_external_python
+  - class-name: airflow.providers.standard.decorators.branch_python.branch_task
+    name: branch
+  - class-name: airflow.providers.standard.decorators.branch_virtualenv.branch_virtualenv_task
+    name: branch_virtualenv
+  - class-name: airflow.providers.standard.decorators.external_python.external_python_task
+    name: external_python
+  - class-name: airflow.providers.standard.decorators.python_virtualenv.virtualenv_task
+    name: virtualenv
+  - class-name: airflow.providers.standard.decorators.sensor.sensor_task
+    name: sensor
+  - class-name: airflow.providers.standard.decorators.short_circuit.short_circuit_task
+    name: short_circuit
```

Comment:
I don't see python (i.e. `@task.python`) in here -- should it be?

Reply:
Yeah I am goin to do that and then i will move it into getattr.

Reply:
Handled in latest commit. It is done through __call__ cos, apparently python does not automatically use __getattr__ for __call__.